### ISSUE TITLE
no auto-repeat on CEC remote

### ISSUE THREAD

**Issue by OttoKerner** [plexinc/plex-media-player#138]
When fixing https://github.com/plexinc/plex-media-player/issues/18 , there might have been a regression:

PMP now ignores prolonged button presses from my CEC remote control e.g. when in the full movie list and holding down the Right arrow key, it only goes one step to the right and then stays there. Same in the jump letters list or simply any list with more than 2 items.

The held down buttons show up (repeated, as they should) in the plexmediaplayer.log but the interface doesn't react to them.
Maybe it now only acts on `<KEY_RELEASE>` or something.

```
2015-10-29 15:45:12 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:12 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:12 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was -1"
2015-10-29 15:45:12 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:12 [ DEBUG ] CachedRegexMatcher.cpp @ 29 - Found cached match for input: "CEC"
2015-10-29 15:45:12 [ DEBUG ] CachedRegexMatcher.cpp @ 29 - Found cached match for input: "KEY_DOWN"
2015-10-29 15:45:12 [ DEBUG ] InputMapping.cpp @ 68 - Mapped: "KEY_DOWN" to action: "down"
2015-10-29 15:45:12 [ DEBUG ] InputComponent.cpp @ 96 - Sending action: "down"
2015-10-29 15:45:12 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:12 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:12 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:12 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:12 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
...
...
2015-10-29 15:45:14 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:14 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:14 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:15 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:16 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key pressed: down (2)
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (0), last was 2"
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): user control pressed (44)
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 295 - CecCommand received  "1 parameter(s) :[0]=2 "
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: key released: down (2)
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 252 - "CecKeyPress : Got key 2 (105), last was 2"
2015-10-29 15:45:17 [ DEBUG ] InputCEC.cpp @ 230 - libCEC DEBUG: >> TV (0) -> Recorder 1 (1): vendor remote button up (8B)
```

**Comment by rbouallou** [comment]
I confirm this.

**Comment by JDIacobbo** [comment]
I can also confirm this on 1.0.3.  I will test on the latest version (1.0.6-RC)

**Comment by purger500** [comment]
I can confirm this on 1.0.5

**Comment by purger500** [comment]
In my log it is not registred as multiple presses but only as one press. Led on my tv is blinking (when i hold down key) but only one press is registered.

**Comment by JDIacobbo** [comment]
I can confirm that this issue still exists in version 1.0.6.

**Comment by tru** [comment]
It's fixed in 1.1

**Comment by JDIacobbo** [comment]
> It's fixed in 1.1

Thanks!  This is the one issue that was preventing me from switching from RasPlex.  Can't wait!
